src/Agent.Sdk/Util/TeeUtil.cs

```diff
@@ -83,12 +83,12 @@ CancellationToken cancellationToken
             string extractedTeePath = Path.Combine(tempDirectory, $"{Guid.NewGuid().ToString()}");
             ZipFile.ExtractToDirectory(zipPath, extractedTeePath);
 
-            debug($"Extracted {zipPath} to ${extractedTeePath}");
+            debug($"Extracted {zipPath} to {extractedTeePath}");
 
             string extractedTeeDestinationPath = GetTeePath();
-            Directory.Move(Path.Combine(extractedTeePath, TeePluginName), extractedTeeDestinationPath);
+            IOUtil.CopyDirectory(Path.Combine(extractedTeePath, TeePluginName), extractedTeeDestinationPath, cancellationToken);
```

Comment:
Should we remove extractedTeePath (since we moved it prevously)?

Reply:
extractedTeePath is located inside tempDirectory, and we delete tempDirectory a couple of lines later